packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -4,139 +4,105 @@ excerpt: >-
 
-> **Note:** For all orgs created after September 2020, this feature is automatically set to the **Enforce and log per client (recommended)** mode.
+### Combination of the line graph and bar graph view
 
-To configure the client-based rate limit for existing orgs:
+The line graph shows usage in the past hour and allows you to focus on a particular point during the hour when an event occurred and caused a spike in the rate limit. The bar graph identifies the top 10 IP addresses that are consuming the most traffic. From there, you can obtain more data, such as observing if there is a sudden spike in usage of one IP while all the other IPs remain relatively normal in usage. You can then identify the offending IP responsible for the spike in traffic for which you should take appropriate action. Alternatively, you may determine that the rate limit may just have a higher rate limit usage and no offending issue, and a higher rate limit is required for the endpoint.
 
-1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limiting** section.
+The line graph can show the number of unique IPs that are creating traffic at a given minute. For example, the line graph can show many users are signing in at the same time, perhaps to attend a company-wide meeting. By comparing the number of unique IPs to the number of unique requests, you can see almost the same number of IPs as the number of requests. This means that there are a lot of IPs creating the traffic and not just one IP consuming all the traffic.
 
-2. Select the type of **Rate limit per client** that you want to implement:
+### Timeline graph view
 
-    * Select **Enforce and log per client (recommended)** to enable client-based rate limit.
+The timeline graph gives you an overview of the usage from the last 28 hours so that you can drill into other time periods from the previous day. This graph enables you to understand whether this rate limit was an abnormal spike in usage compared to fairly regular usage up to that point in time.
 
-    * Select **Log per client** to enable the client-based rate limit in preview mode. In **Log per client** mode, the rate limit is based on the org-wide rate limit values, but client-specific rate limit error information is logged as System Log events. By analyzing these System Log events, you can determine if the client-based rate limit is effective for you.
+### Bar graph view
 
-    * Select **No action** to disable the client-based rate limit.
+The bar graph compares a rogue IP or App with other IPs or Apps. For example, the bar graph can show a single rogue IP that is responsible for a spike in traffic that is displayed in the line graph. It can also show many users that are signing in at the same time, perhaps to attend a company-wide meeting or to check out a special offer that was just delivered in a mass email to a company's clients.
```

Comment:
```suggestion
The bar graph compares a rogue IP or App with other IPs or Apps. For example, the bar graph can show a single rogue IP that is responsible for a spike in traffic that appears in the line graph. It can also show many users that are signing in at the same time, perhaps to attend a company-wide meeting or to check out a special offer that was just delivered in a mass email to a company's clients.
```